src/client/gl-base-technique.cc

```diff
@@ -125,9 +148,26 @@ GlBaseTechnique::Render(Camera* camera)
       auto mvp_location = glGetUniformLocation(program->program_id(), "mvp");
       glUniformMatrix4fv(mvp_location, 1, GL_FALSE, (float*)&camera->vp);
 
+      GLenum texture_target = 0;
+      if (texture) {
+        switch (texture->target()) {
+          case TextureTarget::kNone:
+            break;
+          case TextureTarget::kImage2D:
+            texture_target = GL_TEXTURE_2D;
+            break;
+        }
+      }
```

Comment:
This is not correct. Could you add `target` argument to bind_texture protocol of both gRPC and zigen-gles32?
And use the value here.